test/benchmark/avro_upsert/create_views/views.sql

```diff
@@ -11,19 +11,19 @@ CREATE SOURCE source_upsertavrotest
 FROM KAFKA BROKER 'kafka:9092'
 TOPIC 'upsertavrotest'
 FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://schema-registry:8081'
-ENVELOPE UPSERT
-FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://schema-registry:8081';
+ENVELOPE NONE;
 
 CREATE VIEW upsertavrotest AS SELECT
     -- cast("BookId" AS bigint) AS "BookId",
     -- cast("SecurityId" AS bigint) AS "SecurityId",
     -- cast("TradingBookId" AS int) AS "TradingBookId",
-    "BookId" AS BookId, -- retaining AS long
-    "SecurityId" AS SecurityId, -- retaining AS long
+    "BookIdUnused" AS BookIdUnused, -- retaining AS long
+    "SecurityIdUnused" AS SecurityIdUnused, -- retaining AS long
     (("Exposure")."Current"."Long2"."Exposure")::float AS "CurrentLongExposure", -- converting long to float
     (("Exposure")."Current"."Short2"."Exposure")::float AS "CurrentShortExposure", -- converting long to float
     (("Exposure")."Target"."Long"."Exposure")::float AS "TargetLongExposure", -- converting long to float
     (("Exposure")."Target"."Short"."Exposure")::float AS "TargetShortExposure" -- converting long to float
-FROM source_upsertavrotest;
+FROM source_upsertavrotest
+WHERE "SecurityIdUnused" = 0;
```

Comment:
as noted in slack this s

Reply:
Why does it need to be chosen on the number of input records?

Reply:
so to do an apples to apples comparison with upsert we need to retain as many records as there are unique keys (since thats how much data the upsert indexes will maintain). but since the number of unique keys in the input and the number of records can both vary independently, and we can only specify with the filter the % of input records we want to keep around, there doesnt seem to be an easy way to set this to a constant

e.g. if there are 100 MM input records but 10 MM unique keys then we would want to keep around 10% of records (something like `BookIdUnused < 10`) but if there were 200 MM input records with the same number of unique keys then we would only want to retain 5% of the input

Reply:
I moved this over to #5592 but I ran the test with 1% of keys, not 5/10%.